javascript/javascript-quiz.md

```diff
@@ -1219,8 +1219,8 @@ console.log(animals.shift()); //Prints jaguar
 
 - [ ] `animals.filter(e => e === "jaguar");`
 - [ ] `animals.reverse();`
-- [x] `animals.shift();`
-- [ ] `animals.pop();`
+- [ ] `animals.shift();`
+- [x] `animals.pop();`
```

Comment:
thanks @promeraq for your PR, but i just checked it in console, as result 
![image](https://user-images.githubusercontent.com/13170022/224565301-fc365d3c-4755-4053-a015-1188e582746b.png)

Reply:
Q93. Which line COULD YOU ADD to this code to print "jaguar" to the console? 
You are just checking the statement in console, but have to add a missing line.

Reply:
you are right, LGTM